openvdb/version.h

```diff
@@ -174,7 +174,7 @@
 
 /// By default, the @b OPENVDB_REQUIRE_VERSION_NAME macro is undefined, and
 /// symbols from the version namespace are promoted to the top-level namespace
-/// so that, for example, @b openvdb::v6_0::io::File can be referred to
+/// so that, for example, @b openvdb::v5_0::io::File can be referred to
```

Comment:
This is to revert this back to the previous version number with the intention that it isn't updated every version change right?

Reply:
> I can see why a patch version might be preferable though because it won't (potentially) require downgrading the version number.

Changing only the patch number also preserves the library namespace, FWIW.  And note that introducing new features, even substantial ones, doesn't necessitate a minor version bump, unless those changes break existing APIs.

> This is to revert this back to the previous version number with the intention that it isn't updated every version change right?

Right, there's no need to keep that comment in sync with the library version.

On the other hand, it _is_ necessary to keep the `vdbnamespace` and `OPENVDB_VERSION_NAME` definitions in sync (in [openvdb/doxygen-config](https://github.com/AcademySoftwareFoundation/openvdb/blob/d7f6c3d9afd74c1a3f585ac4ee4dcfd526dab809/openvdb/doxygen-config#L44)).